content/applications/sales/crm/track_leads/marketing_attribution.rst

```diff
@@ -0,0 +1,142 @@
+graph visualization displays the number of leads generated, by month and by sales team, with each
+sales team attributed to its own color per bar (month) shown.
+
+Switch the view to the :icon:`oi-view-list` :guilabel:`(List)` option, by clicking the respective
+icon located at the top-right of the dashboard.
+
+.. image:: marketing_attribution/list-view-button.png
+   :align: center
+   :alt: Click the button with four horizontal lines on the top right of the Leads Analysis page.
+
+Add UTM parameters
+==================
+
+*Urchin Tracking Modules (UTMs)* are snippets of text embedded in URLs that allow for data about
+visitors to be tracked. This includes parameters relating to how a visitor reached the link, such as
+the type of website visited and marketing campaign the visit came from.
+
+Odoo can use these UTMs as parameters in the marketing attribution report to track the metrics and
+performance of marketing campaigns.
+
+Create UTMs
+-----------
+
+The :doc:`link tracker <../../../websites/website/reporting/link_tracker>` in Odoo can be used to
+create and configure UTMs.
+
+UTMs can also be automatically generated by :doc:`Email Marketing
+<../../../marketing/email_marketing>` and :doc:`Marketing Automation
+<../../../marketing/marketing_automation>` apps.
+
+The UTM parameters used in a marketing attribution report are *Medium*, *Source*, and *Campaign* in
+descending order of coverage.
+
+- *Medium* is the UTM with the widest coverage, such as social media.
+- *Source* is more narrow and covers parameters such as the specific social media platform.
+- *Campaign* is the narrowest and can track specific marketing campaigns.
+
+Create reports
+==============
+
+To start creating a report, click the :icon:`fa-caret-down` :guilabel:`(down caret)` to the right of
+the search bar to see the list of filtering and grouping parameters.
+
+:guilabel:`Filters`, located in the left column of the search options, can be used to keep only
+the results that fit the filter. For example, selecting the :guilabel:`Won` filter will only
+show won leads in the attribution report. :guilabel:`Group By`, found in the middle column, is
+used to organize the results into groups and can be used with or without filters.
+
+.. image:: marketing_attribution/search-results-multiple-options.png
+   :align: center
+   :alt: Select any number of filters and groups in the search options.
+
+.. tip::
+    Setting multiple Group By options will create nested groups according to which is selected
+    first. For example, selecting :guilabel:`Medium` followed by :guilabel:`Source` and then
+    ::guilabel:`Campaign` in :guilabel:`Group By` will sort all results *first* by medium, *then* by
+    the specific sources in each medium, followed by the campaigns in each source.
+
+    This can be verified by looking at the direction of the carat in the group tile in
+    the search bar.
```

Comment:
```suggestion
    This can be verified by looking at the direction, and order, of the selections in the group tile that appears in
    the :guilabel:`Search...` bar.
```